src/state.rs

```diff
@@ -0,0 +1,13 @@
+use cosmwasm_schema::cw_serde;
+use cosmwasm_std::Uint64;
+use cw_storage_plus::Map;
+
+#[cw_serde]
+pub struct StoredData {
+    pub data: String,
```

Comment:
This is redundant, as the hash of the data is currently being used as key. But anyway, since it's there, I guess it's not a big deal to return it anyway.